ethcore/sync/src/chain/mod.rs

```diff
@@ -88,6 +88,7 @@
 //! All other messages are ignored.
 
 mod handler;
+pub mod syncpacketid;
```

Comment:
I'm not a fan of the name, why not `sync_packets`?

Reply:
Mmmm, no particular reason, I'll change it.